src/tests/ReactElementAdapter.spec.jsx

```diff
@@ -278,7 +278,8 @@ describe('ReactElementAdapter', () => {
         it('returns the expect.it function for an expect.it content assertion', () => {
 
             const component = <span>{expect.it('to be a string')}</span>;
-            expect(adapter.getChildren(component), 'to equal', [ expect.it('to be a function')] )
```

Comment:
Ah, I see.  This should be 

```js
expect(adapter.getChildren(component), 'to satisfy', [ expect.it('to be a function')])
```

This test was wrong - I'm amazed it worked in any version, now I see it!